src/restful/controllers/authController.js

```diff
@@ -213,6 +224,7 @@ class AuthController {
       const nonSensitiveData = {
         first_name: userData.first_name,
```

Comment:
This function is designed solely for verifying OTP and does not interact with nonSensitiveData. Therefore, it is advised not to make any modifications to it